cypress/component/Calendar.cy.js

```diff
@@ -4,13 +4,71 @@ import localeDa from "dayjs/locale/da";
 import Calendar from "../../src/calendar/calendar";
 
 describe("Calendar", () => {
-  it("Calendar multiple days", () => {
+  it("Calls slideDone correctly", () => {
     const mock = {
       slideDone: (arg) => {
         return arg;
       },
     };
     cy.stub(mock, "slideDone").as("slideDoneStub");
+
+    cy.mount(
+      <div className="slide" id="SLIDE_ID">
+        <Calendar
+          content={{
+            duration: 500,
+            layout: "multipleDays",
+            hasDateAndTime: true,
+            title: "Kalender",
+            subTitle: "Underoverskrift",
+            resourceAvailableText: "Lokalet er frit.",
+            displayHeaders: true,
+            resourceUnavailableText: "Det er optaget",
+            image: [],
+            footerText: "Se mere på localhost/events",
+            fontSize: "font-size-m",
+          }}
+          slide={{
+            id: "slide2-calendar-multiple-days",
+            type: "calendar",
+            themeFile: "themes/dokk1.css",
+            feedData: [],
+            mediaData: {},
+            content: {
+              duration: 500,
+              layout: "multipleDays",
+              hasDateAndTime: true,
+              title: "Kalender",
+              subTitle: "Underoverskrift",
+              resourceAvailableText: "Lokalet er frit.",
+              displayHeaders: true,
+              resourceUnavailableText: "Det er optaget",
+              image: [],
+              footerText: "Se mere på localhost/events",
+              fontSize: "font-size-m",
+            },
+          }}
+          run={new Date().toISOString()}
+          slideDone={mock.slideDone}
+          executionId="SLIDE_ID"
+        />
+      </div>
+    );
+
+    cy.get("@slideDoneStub").should("not.be.called");
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(500);
+    cy.get("@slideDoneStub").should("be.called");
+  });
+
+  it("Calendar multiple days", () => {
+    const timestamp = 60 * 60 * 1000 * 8;
```

Comment:
Add comment explaining the choice of math :-)

Consider changing `timestamp` to a more descriptive name